packages/react-native-sdk/src/components/participants/ParticipantView.tsx

```diff
@@ -273,7 +273,7 @@ export const ParticipantView = (props: ParticipantViewProps) => {
                 <VideoSlash color={theme.light.error} />
               </View>
             )}
-            {participant.pinnedAt && (
+            {participant.pin && participant.pin.isLocalPin && (
```

Comment:
Is there a way we can simplify this check? 🤔

Reply:
I'm avoiding using `?.` and `??` operators as I think they make the code less understandable as they increase its cognitive complexity.

Do you think it adds more value if I shorten this expression?

Reply:
That's true, but maybe we can extract this check into a variable and then use the variable here. Makes it easier to understand.

Something like:
const isPinnedEnabled = participant.pin && participant.pin.isLocalPin;